addons/civ_placement/tests/auto_clusterGeneration.sqf

```diff
@@ -1,6 +1,6 @@
 // ----------------------------------------------------------------------------
 
-#include "\x\alive\addons\civ_placement\script_component.hpp"
+#include <\x\alive\addons\civ_placement\script_component.hpp>
 SCRIPT(auto_clusterGeneration);
 
 //execVM "\x\alive\addons\civ_placement\tests\auto_clusterGeneration.sqf"
```

Comment:
These changes shouldn't be here.

Reply:
I will correct mistakes, thank you for helping me along the way.